photon-client/src/integTest/scala/com/linkedin/photon/ml/estimators/GameEstimatorTest.scala

```diff
@@ -38,20 +47,42 @@ class GameEstimatorTest extends SparkTestUtils with GameTestUtils {
   @Test
   def simpleTest(): Unit = sparkTest("simpleTest") {
 
+    val featureShardId = "shard1"
     val params = new GameParams
     params.taskType = TaskType.LINEAR_REGRESSION
     params.normalizationType = NormalizationType.STANDARDIZATION
     params.fixedEffectDataConfigurations =
-      Map("global" -> FixedEffectDataConfiguration.parseAndBuildFromString("shard1,1"))
+      Map("global" -> FixedEffectDataConfiguration.parseAndBuildFromString(s"$featureShardId,1"))
 
-    val logger = new PhotonLogger(s"${params.outputDir}/log", sc)
-    val estimator = new GameEstimator(sc, params, logger)
+    val (estimator, logger) = createEstimator(params, "simpleTest")
+    val (nSamples, nDimensions) = (10, 3)
```

Comment:
I'd prefer these two each on their own line - I don't like defining multiple vals on one line in Scala unless it's through an unapply method